content/docs/started/getting-started-gke.md

```diff
@@ -352,16 +352,40 @@ means the GKE nodes won't have public IP addresses. This can improve security by
 access to nodes. Removing IP addresses means external services (including GitHub, PyPi, DockerHub etc...) won't be accessible
 from the nodes. Google services (e.g. GCR) are still accessible.
 
-1. Enable private clusters in `${KFAPP}/gcp_configs/cluster-kubeflow.yaml` by updating the line
+1. Enable private clusters in `${KFAPP}/gcp_configs/cluster-kubeflow.yaml` by updating the following two parameters
 
     ```
     privatecluster: true
+    gkeApiVersion: v1beta1
     ```
 1. Create the deployment
 
     ```
     cd ${KFAPP}
-    ${KUBEFLOW_REPO}/scripts/kfctl.sh apply all 
+    ${KUBEFLOW_REPO}/scripts/kfctl.sh apply platform
+    ```
+
+1. Update the various ksonnet components to use gcr.io images instead of dockerhub images
+
+    ```
+    cd ${KFAPP}/ks_app
+    ${KUBEFLOW_REPO}/scripts/gke/use_gcr_for_all_images.sh
+    ```
+
+1. Remove components which are not useful in private clusters
+
+    ```
+    cd ${KFAPP}/ks_app
+    ks component rm cloud-endpoints
+    ks component rm cert-manager
+    ks component rm iap-ingress
```

Comment:
Doesn't IAP still work with private clusters?If not what's the issue and what's the alternative ingress story?

Reply:
I haven't tried IAP because cert-manager cannot obtain certificate from letsencrypt in a private cluster. I can play around with it and send a follow-up PR with how to make IAP work.

Reply:
As of this doc, the ingress story is to do port-forwarding